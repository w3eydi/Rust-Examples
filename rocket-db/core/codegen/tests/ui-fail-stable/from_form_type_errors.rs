../ui-fail/from_form_type_errors.rs