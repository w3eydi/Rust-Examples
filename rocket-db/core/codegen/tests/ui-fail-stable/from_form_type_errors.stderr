error[E0277]: the trait bound `Unknown: FromFormField<'_>` is not satisfied
 --> $DIR/from_form_type_errors.rs:7:12
  |
7 |     field: Unknown,
  |            ^^^^^^^ the trait `FromFormField<'_>` is not implemented for `Unknown`
  |
  = note: required because of the requirements on the impl of `FromForm<'r>` for `Unknown`

error[E0277]: the trait bound `Foo<usize>: FromFormField<'_>` is not satisfied
  --> $DIR/from_form_type_errors.rs:14:12
   |
14 |     field: Foo<usize>,
   |            ^^^ the trait `FromFormField<'_>` is not implemented for `Foo<usize>`
   |
   = note: required because of the requirements on the impl of `FromForm<'r>` for `Foo<usize>`
